lib/http/response.rb

```diff
@@ -125,6 +129,15 @@ def cookies
       end
     end
 
+    def chunked?
+      encoding = headers.get(Headers::TRANSFER_ENCODING)
+      case encoding
+        when String then encoding == CHUNKED
+        when Enumerable then encoding.include?("chunked")
+        else false
+      end
```

Comment:
In fact the only thing that annoys me in "complex" logic of pseudo-code above is how it will work with persistent connection (AKA `Keep-Alive`) :D Probably we can actually implement it and raising `ResponseError` if connection is persistnet and transfer encoding is chunked (but it's not final encoding) - I believe it's the most retarded and rare case ever :D